cmd/pinniped-proxy/src/service.rs

```diff
@@ -1,7 +1,17 @@
 use std::convert::Infallible;
 
 use hyper::{Body, Request, Response};
+use log::info;
 
-pub async fn proxy(_: Request<Body>) -> Result<Response<Body>, Infallible> {
-    Ok(Response::new(Body::from("pinniped-proxy stub\n")))
+use crate::logging;
+
+pub async fn proxy(req: Request<Body>) -> Result<Response<Body>, Infallible> {
+    let log_data = logging::request_log_data(&req);
+
+    // TODO: actual proxying to happen here.
+    let response = Response::new(Body::from("pinniped-proxy stub\n"));
+
+    info!("{}", logging::response_log_data(&response, log_data));
```

Comment:
Small update to our proxy stub function so that it now records the data for logging before and after the (soon-to-be) actual proxying of the request, and includes the log at the info level.